### PR TITLE
feat(dockerfile): Add Dockerfile

### PR DESCRIPTION
**Pull request by KasnocknDave**
**Issue link:** [Issue title](Issue Link) 

### Code Review Checklist

- [ ] Code changes follow CONTRIBUTING.md
- [ ] README.md is up to date and maintained
- [ ] New code is covered with test cases

### CODE REVIEW

File: apps/platform/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+ENV NVM_DIR=/root/.nvm
+RUN . "$NVM_DIR/nvm.sh" && nvm install 19
+RUN . "$NVM_DIR/nvm.sh" && nvm use 19
+RUN . "$NVM_DIR/nvm.sh" && nvm alias default
+ENV PATH="/root/.nvm/versions/node/v19.9.0/bin/:${PATH}"
+
+RUN node --version
+
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
```
Comment (flobauer) [inline apps/platform/Dockerfile:17]: I like how this is setup, well done!

File: apps/platform/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+WORKDIR /app
+COPY . .
+
+RUN apt update && apt install git -y
+
+RUN composer update && composer install
+
+RUN npm install & npm run build
+
+CMD ["php", "artisan", "serve", "--host=0.0.0.0"]
```
Comment (flobauer) [inline apps/platform/Dockerfile:28]: NGINX avoided ftw

**Review by flobauer — APPROVED** [review]
Great job

File: apps/platform/Dockerfile (hunk 2 above)
Comment (KasnocknDave) [reply apps/platform/Dockerfile:28]: too much development complexity

Comment (KasnocknDave) [reply apps/platform/Dockerfile:28]: won't implement

**Comment by flobauer** [comment]
@tosh wdyt?

**Pull request merged**